.github/workflows/lint-test.yml

```diff
@@ -1,71 +1,75 @@
 name: Lint & Test
 
 on:
-    push:
-        branches:
-            - main
-    pull_request:
+  push:
+    branches:
+      - main
+  pull_request:
 
 jobs:
-    lint-test:
-        runs-on: ubuntu-latest
-        env:
-            # Disable pip cache and do user installs by default.
-            PIP_NO_CACHE_DIR: false
-            PIP_USER: 1
+  lint-test:
+    runs-on: ubuntu-latest
+    env:
+      # Disable pip cache and do user installs by default.
+      PIP_NO_CACHE_DIR: false
+      PIP_USER: 1
 
-            POETRY_VIRTUALENVS_CREATE: false
+      POETRY_VIRTUALENVS_CREATE: false
 
-            PYTHONUSERBASE: ${{ github.workspace }}/.cache/py-user-base
-            PRE_COMMIT_HOME: ${{ github.workspace }}/.cache/pre-commit-cache
+      PYTHONUSERBASE: ${{ github.workspace }}/.cache/py-user-base
+      PRE_COMMIT_HOME: ${{ github.workspace }}/.cache/pre-commit-cache
 
-        steps:
-            - name: Add custom PYTHONUSERBASE to PATH
-              run: echo '${{ env.PYTHONUSERBASE }}/bin/' >> $GITHUB_PATH
+    steps:
+      - name: Add custom PYTHONUSERBASE to PATH
+        run: echo '${{ env.PYTHONUSERBASE }}/bin/' >> $GITHUB_PATH
 
-            - name: Checkout repository
-              uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v2
 
-            - name: Setup Python
-              id: python
-              uses: actions/setup-python@v2
-              with:
-                  python-version: "3.9"
+      - name: Setup Python
+        id: python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
 
-            - name: Python cache
-              uses: actions/cache@v2
-              id: python_cache
-              with:
-                  path: ${{ env.PYTHONUSERBASE }}
-                  key: "python-0-${{ runner.os }}-${{ env.PYTHONUSERBASE }}-\
-                      ${{ steps.python.outputs.python-version }}-\
-                      ${{ hashFiles('./pyproject.toml', './poetry.lock') }}"
+      - name: Setup Node.js
```

Comment:
Please include the comment here which explains why it is necessary to explicitly specify a node version.